monkey/infection_monkey/post_breach/actions/modify_shell_startup_files.py

```diff
@@ -0,0 +1,32 @@
+from common.data.post_breach_consts import POST_BREACH_SHELL_STARTUP_FILE_MODIFICATION
+from infection_monkey.post_breach.pba import PBA
+from infection_monkey.utils.shell_startup_files_modification import\
+    get_commands_to_modify_shell_startup_files
+from infection_monkey.utils.environment import is_windows_os
+
+
+class ModifyShellStartupFiles(PBA):
+    """
+    This PBA attempts to modify shell startup files,
+    i.e. ~/.bashrc, ~/.profile, ~/.bash_profile in linux,
+    and profile.ps1 in windows.
+    """
+
+    def __init__(self):
+        super(ModifyShellStartupFiles, self).__init__(name=POST_BREACH_SHELL_STARTUP_FILE_MODIFICATION)
+
+    def run(self):
+        (cmds_for_linux, shell_startup_files_for_linux), windows_cmds = get_commands_to_modify_shell_startup_files()
+
+        if is_windows_os():
+            super(ModifyShellStartupFiles, self).__init__(name=POST_BREACH_SHELL_STARTUP_FILE_MODIFICATION,
+                                                          linux_cmd='',  # windows so won't matter
+                                                          windows_cmd=windows_cmds)
+            super(ModifyShellStartupFiles, self).run()
+        else:
+            for shell_startup_file in shell_startup_files_for_linux:
+                linux_cmds = ' '.join(cmds_for_linux).format(shell_startup_file)
```

Comment:
`run()` method should *run* post breach actions, not craft cmd's. Create a separate method that returns a list of all linux cmd's. Move this line to that method.